docs/development/design/organizations.rst

```diff
@@ -0,0 +1,145 @@
+Organizations
+=============
+
+Currently we don't support organizations in the community site
+(a way to group different projects),
+we only support individual accounts.
+
+Several integrations that we support have organizations,
+where users group their repositories.
+
+Why move organizations in the community site?
+---------------------------------------------
+
+We support organizations in the commercial site,
+having no organizations in the community site makes the code maintenance difficult.
+Having organizations in the community site will make the differences between both more easy to manage.
+
+Users from the community site can have organizations in external sites from where we import their projects
+(like GitHub, Gitlab).
+Currently users have all projects from different organizations in their account.
+Having not a clear way to group/separate those.
+
+There is also a product decision.
+Supporting organizations only on code,
+but enable the feature only on the commercial site.
+
+How are we going to support organizations?
+------------------------------------------
+
+Currently only users can own projects.
+With organizations this is going to change to: 
+Users and organizations can own projects.
+
+With this, the migration process would be straightforward for the community site.
+
+For the commercial site we still have a decision.
+Are we going to support users and organizations to own projects?
+Or just organizations?
+
+What features of organizations are we going to support?
+-------------------------------------------------------
+
+We have this features present in the commercial site:
+
+- Owners
+- Teams
+- Permissions
+- Subscriptions
+
+Owners should be included to represent owners of the current organization.
+
+Teams, this is also handy to manage access to different projects under the same organization.
+
+Permissions,
+currently we have two type of permissions for teams: admin and read only.
+Read only permissions doesn't make sense in the community site since we only support public projects/versions
+(we do support private versions now, but we are planning to remove those).
+So, we should only support admin permissions for teams.
```

Comment:
These permissions are basically a field: `Team.access`. It's not another entity by itself.

Reply:
Agreed that teams should only support admin currently. 👍

Reply:
What about API access? Don't we want to be able to add read-only users to the organization for API access? If we can avoid creating differences between the installations, that is a large benefit.

Reply:
The current idea for api access is to create scope based tokens.